python/lsst/dm/csc/base/message_director.py

```diff
@@ -514,72 +615,93 @@ async def transmit_startIntegration(self, data):
             report = f"No ack response from at archive controller"
 
             waiter = Waiter(evt, self.parent, self.ack_timeout)
-            task = asyncio.create_task(waiter.pause(code, report))
+            asyncio.create_task(waiter.pause(code, report))
 
     async def process_xfer_params_ack(self, msg):
-        """ Handle xfer_params_ack message
+        """Handle xfer_params_ack message
+
+        Parameters
+        ----------
+        msg : `dict`
+            Contents of the xfer_params message
         """
         ack_id = msg["ACK_ID"]
         LOGGER.info(f"startIntegration ack_id = {ack_id} received")
-        evt = await self.clear_event(ack_id)
+        await self.clear_event(ack_id)
 
     #
     # endReadout
     #
-    async def transmit_endReadout(self, data):
+    async def transmit_endReadout(self, msg):
         """transmit endReadout to the forwarder
-        @param data: the contents of the endReadout CSC message
+
+        Parameters
+        ----------
+        msg : `dict`
```

Comment:
This is not a `dict`. It is salObj.[DataType](https://ts-salobj.lsst.io/py-api/lsst.ts.salobj.ConfigurableCsc.html?highlight=datatype). For example, you can't access element from `msg` with `msg['key']`.